pytest_mock.py

```diff
@@ -10,7 +10,7 @@
 version = '1.2'
 
 
-class MockFixture(object):
+class MockFixture:
```

Comment:
Oh right. I tried that first as a fix and forgot to revert it.

On the other hand being an old-style class doesn't really matter for `MockFixture` because it doesn't have any comparison operators or meta-classes, which is where old-style and new-style classes differ. Do you rather me revert that?